java/java-jwt/security/jwt-hardcode.java

```diff
@@ -48,6 +48,7 @@ public static void main( String[] args )
 
     public void bad2() {
         try {
+            // ruleid: java-jwt-hardcoded-secret
             Algorithm algorithm = Algorithm.HMAC256(secret);
```

Comment:
Shouldn't have this be matching before? Maybe I haven't looked into this carefully enough, but it feels like it should have matched already.